draft-ietf-scitt-architecture.md

```diff
@@ -771,7 +770,7 @@ When a Receipt is included in a Signed Statement a Transparent Statement is prod
 
 Receipts are based on Signed Inclusion Proofs as described in COSE Receipts {{-COMETRE}} that also provides the COSE header parameter semantics for label TBD_0.
 
-The Registration time is recorded as the timestamp when the Transparency Service added this Signed Statement to its Append-only Log.
+The Registration time is recorded as the timestamp when the Transparency Service added the Signed Statement to its Verifiable Data Structure.
```

Comment:
```suggestion
The Registration time is recorded as the timestamp when the Transparency Service added the Signed Statement to its Append-only Log.
```

Reply:
This would introduce the possibility of the VDS time and append only log time being different?
... also begs the question of "where is registration time" recorded.

Reply:
Yes. we specicically included a record of time in the VDS so that it can be authorative. We do record a time at which the ts first recieves the registration request. But we see that firmly as part of the "adjacent service" data schema (not scitt). 

We ensure that "time of comitment to the vds" is consistent with the order of statements in the log and see this as very much the business of the TS and its VDS

OR remove the requirement for the registration time to be recorded altogether

Reply:
CCF records it in the CWT_Claims.iat of the Receipt, because it seemed like the right place for it. I think what we are saying is that the VDS inclusion is what matters (that's where the proofs come from), and how the sequence of Signed Statements/log is stored is the business of an Adjacent service and comparatively less important, so it's not obvious that it deserves its own timestamp?

Reply:
Following discussion, I agree this change is not helpful.